### ISSUE TITLE
Config could not be loaded after i update docker-sync to 0.3.4

### ISSUE THREAD

**Issue by suhanda** [EugenMayer/docker-sync#305]
i got miss configuration on `xargs -I -n1 docker-sync sync -n php-lumio-sync --config=''`  
because of this docker-sync not working.

```fswatch --exclude='.*/.git' --exclude='.*/node_modules' --exclude='.*/bower_components' --exclude='.*/sass-cache' --exclude='.*/.sass-cache' --exclude='.*/.sass-cache' --exclude='.coffee' --exclude='.scss' --exclude='.sass' --exclude='.gitignore' --exclude='.*/app/vendor/*' --exclude='app/common/migrations/*' --exclude='app/common/migrations-mongo/*' -orIE --event AttributeModified --event Created --event Link --event MovedFrom --event MovedTo --event Renamed --event Removed --event Updated -v '/Users/dedisuhanda/works/goyo/'  | xargs -I -n1 docker-sync sync -n php-lumio-sync --config=''```

**Comment by EugenMayer** [comment]
i have no idea what you just told me. Could you rephrase ?

that command is from the fswatch watcher strategy and why is it failing? Why is it not working. Comeone, please provide informations, i am not a magician

**Comment by suhanda** [comment]
Sorry, my english bad, 

I got that command when i run `docker-sync-stack` with verbose mode true, i assume because command with` --config = '' ` ` docker-sync` could not loaded config file because parameter `--config` is empty. 
this error message i'v got from `docker-sync` : 
`error  Config could not be loaded from  - it does not exist`

**Comment by EugenMayer** [comment]
@ignatiusreza could that be part of our configuration rewrite?

@suhanda please paste your output as a sequence so we can have an idea when this happens. do you see any stacktrace?

**Comment by suhanda** [comment]
This all output i've got: 

```
note:  You can also run docker-sync in the background with docker-sync --daemon
          ok  Starting rsync
          ok  php-lumio-sync container not running
          ok  starting php-lumio-sync container
     command  docker start php-lumio-sync
          ok  php-lumio-sync: starting initial sync of /Users/dedisuhanda/works/goyo/
     command  rsync --exclude='node_modules/' --exclude='bower_components/' --exclude='vendor/bin/' --exclude='provision/' --exclude='scripts/' --exclude='datalog/' --exclude='.sass-cache/' --exclude='sass/' --exclude='sass-cache/' --exclude='app/common/migrations/' --exclude='app/common/migrations-monggo/' --exclude='.git/' --exclude='web/assets/*' --exclude='runtime/*' --exclude='*.md' --exclude='composer.json' --exclude='composer.lock' --exclude='package.json' --exclude='Gruntfile*' --exclude='*.coffee' --exclude='*.scss' --exclude='*.sass' --exclude='*.less' --exclude='*.log' --exclude='docker-compose.yml' --exclude='docker-compose-native.yml' --exclude='docker-compose.native.yml' --exclude='run.sh' --exclude='shell.sh' --exclude='influx.sh' --exclude='mysql.sh' --exclude='mongo.sh' --exclude='redis.sh' --exclude='yii.sh' --exclude='clean.sh' --exclude='migration.sh' --exclude='CHANGELOG.txt' --exclude='.idea' --exclude='.DS_Store' --exclude='.vscode' --exclude='*.sublime-project' -ap -v '/Users/dedisuhanda/works/goyo/' rsync://localhost:10872/volume
          ok  Synced /Users/dedisuhanda/works/goyo/
      output  sending incremental file list

sent 624,568 bytes  received 5,678 bytes  1,260,492.00 bytes/sec
total size is 217,105,594  speedup is 344.48
     success  Rsync server started
     command  rsync --exclude='node_modules/' --exclude='bower_components/' --exclude='vendor/bin/' --exclude='provision/' --exclude='scripts/' --exclude='datalog/' --exclude='.sass-cache/' --exclude='sass/' --exclude='sass-cache/' --exclude='app/common/migrations/' --exclude='app/common/migrations-monggo/' --exclude='.git/' --exclude='web/assets/*' --exclude='runtime/*' --exclude='*.md' --exclude='composer.json' --exclude='composer.lock' --exclude='package.json' --exclude='Gruntfile*' --exclude='*.coffee' --exclude='*.scss' --exclude='*.sass' --exclude='*.less' --exclude='*.log' --exclude='docker-compose.yml' --exclude='docker-compose-native.yml' --exclude='docker-compose.native.yml' --exclude='run.sh' --exclude='shell.sh' --exclude='influx.sh' --exclude='mysql.sh' --exclude='mongo.sh' --exclude='redis.sh' --exclude='yii.sh' --exclude='clean.sh' --exclude='migration.sh' --exclude='CHANGELOG.txt' --exclude='.idea' --exclude='.DS_Store' --exclude='.vscode' --exclude='*.sublime-project' -ap -v '/Users/dedisuhanda/works/goyo/' rsync://localhost:10872/volume
          ok  Synced /Users/dedisuhanda/works/goyo/
      output  sending incremental file list

sent 624,516 bytes  received 5,626 bytes  420,094.67 bytes/sec
total size is 217,105,594  speedup is 344.53
          ok  Cli mode is auto, selecting .. 
          ok  .. docker-sync 
     success  Starting to watch /Users/dedisuhanda/works/goyo/ - Press CTRL-C to stop
     command  fswatch --exclude='.*/.git' --exclude='.*/node_modules' --exclude='.*/bower_components' --exclude='.*/sass-cache' --exclude='.*/.sass-cache' --exclude='.*/.sass-cache' --exclude='.coffee' --exclude='.scss' --exclude='.sass' --exclude='.gitignore' --exclude='.*/app/vendor/*' --exclude='app/common/migrations/*' --exclude='app/common/migrations-mongo/*' -orIE --event AttributeModified --event Created --event Link --event MovedFrom --event MovedTo --event Renamed --event Removed --event Updated -v '/Users/dedisuhanda/works/goyo/'  | xargs -I -n1 docker-sync sync -n php-lumio-sync --config=''
          ok  Found implicit docker-compose-dev.yml and using it from /Users/dedisuhanda/works/goyo/docker-compose-dev.yml
          ok  starting compose
Starting lumio_redis
Starting lumio_influx
Starting lumio_mysql
Starting lumio_mongodb
Sync php-lumio-sync         error  Config could not be loaded from  - it does not exist
Sync php-lumio-sync         error  Config could not be loaded from  - it does not exist
Sync php-lumio-sync         error  Config could not be loaded from  - it does not exist
```

**Comment by EugenMayer** [comment]
please remove the sycn/watch excludes and try once again, please also remove dest: '/var/www/' since its no longer supported, since we have named volumes

**Comment by suhanda** [comment]
https://github.com/EugenMayer/docker-sync/issues/305#issuecomment-297020967
I am still got same error message

**Comment by ignatiusreza** [comment]
I think I know what's going wrong..I'll take a look first thing in the morning..

**Comment by EugenMayer** [comment]
Please try the docker-sync-boilerplate with rsync https://github.com/EugenMayer/docker-sync-boilerplate/tree/master/rsync - does this work?

**Comment by suhanda** [comment]
@EugenMayer same error message `error  Config could not be loaded from  - it does not exist`

**Comment by EugenMayer** [comment]
@ignatiusreza it does happen on sync, so on host file change

**Comment by EugenMayer** [comment]
Fixed with https://github.com/EugenMayer/docker-sync/commit/2bf2a6d0348f85e90924821b4b0e9940c0372292 .. deployed with 0.3.5

**Comment by suhanda** [comment]
@EugenMayer it's working now.
Thank you. you save my work
